app/views/admin/works/index.html.erb

```diff
@@ -89,7 +89,10 @@
       <th class="datestamp"><%= sort_link(@q, :created_at, "Created", page: nil) %></th>
       <th class="datestamp"><%= sort_link(@q, :updated_at, "Last Modified", page: nil) %></th>
       <th class="action"></th>
-      <th>Cart</th>
+
+      <% if can?(:update, Kithe::Model) %>
+        <th>Cart</th>
+      <% end %>
```

Comment:
1. What makes this the right privilge to check for adding/removing someting from your personal cart? And why do we need a privilege to check, is there anyone who has access to this page who does NOT have access to a Cart?

Reply:
Oh, I logged in as one of the "staff viewer" users and noticed  the cart column is entirely empty for them. But this might belong in another PR.